integration_test/README.md

```diff
@@ -0,0 +1,95 @@
+# Integration Tests
+
+Integration tests can be run in develop (local) mode or in CI environment for validating VAL components in Dapr/Kubernetes environment.\
+SeatService is running in simulated CAN mode `CAN="cansim"` and SeatService client is used to start seat movement.
+
+## Integration Test overview
+
+- `integration_test/test_feeder.py`:\
+    This test covers feedercan datapoints. It checks if they are registered and also that some datapoints have changing values.\
+    **NOTE:** If feedercan default config is changed, make sure those changes are reflected in feeder test.
+
```

Comment:
Should be move to feeder repo

Reply:
You can remove it, I may add it back when I have some integration test strategy.. 
I really like to be able to start all components in a single vscode project (via tasks), but maybe it won't be feasible now

Reply:
For me that are two separate things:
I also like being able to start broker, CAN feeder, and services in the services repo for manual testing. 
But automated integration tests should be located "where they belong".